open-metadata-implementation/user-interfaces/presentation-server/cra-client/src/Frame.js

```diff
@@ -1,15 +1,17 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 /* Copyright Contributors to the ODPi Egeria project. */
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 import Search20 from "@carbon/icons-react/lib/search/20";
 import Notification20 from "@carbon/icons-react/lib/notification/20";
 import AppSwitcher20 from "@carbon/icons-react/lib/app-switcher/20";
+import { Logout32 } from "@carbon/icons-react";
 import HeaderContainer from "carbon-components-react/lib/components/UIShell/HeaderContainer";
 import { Content } from "carbon-components-react/lib/components/UIShell";
 import { Link, Route, Switch } from "react-router-dom";
 import Egeriawhite110 from "./images/Egeria_logo_white_110";
```

Comment:
as above - can we reuse these icons from egeria core

Reply:
Do you mind referencing where these are stored in the egeria core? I am not super familiar with the rest of the project yet. I see `assets/img`, but that directory does not seem to have these assets.

Reply:
good question ... my thought was to collect them together - so add if needed, but needs further discussion so let's consider in #3576